tutor/static/css/base.css

```diff
@@ -169,7 +169,7 @@ a:hover {
 
 
 /*MOBILE STYLING */
-@media screen and (max-width: 768px) {
+@media screen and (max-width: 766px) {
```

Comment:
The `visible-xs` attribute on bootstrap is set to 768px. So with this change, if you have a screen width of 767px then you'll see the icon banner but none of the other styling you have here.